en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md

```diff
@@ -176,11 +210,42 @@ This section involves setting up the Traffic Manager node(s) and enabling it to
 
 [![Traffic Manager Connections]({{base_path}}/assets/img/setup-and-install/traffic-manager-connections.png)]({{base_path}}/assets/img/setup-and-install/traffic-manager-connections.png)
 
+1.  Open the `<API-M_HOME>/repository/conf/deployment.toml` file in the Traffic Manager node and change as following to point to Key Manager nodes.
+
+    ``` toml tab="Key Manager with HA"
+        [apim.key_manager]
+        service_url = "https://[Key-Manager-LB-host]/services/"
+        username = "$ref{super_admin.username}"
+        password = "$ref{super_admin.password}"
+    ```
+        
+    ``` toml tab="Single Key Manager"
+        [apim.key_manager]
+        service_url = "https://[Key-Manager-host]:${mgt.transport.https.port}/services/"
+        username = "$ref{super_admin.username}"
+        password = "$ref{super_admin.password}"
+    ```
+##### Configuring High Availability (HA) for the Traffic Manager.
```

Comment:
looks like this line is redundant or shall we make it bold to hight light it as a sub topic

Reply:
@shaniR wdyt ?

Reply:
Since it's a heading, I think it's fine.